### PR TITLE
[FEATURE] Webflow Update Collection Item action - Add Custom Fields p…

### PR DESCRIPTION
**Pull request by luancazarine**
…rop #6579

  - Add customFields prop

### <samp>🤖 Generated by Copilot at 6f6edad</samp>

Enhanced the `update-collection-item` action for Webflow with a new prop and a new response format. Changed the `updateItem` method in `webflow.app.mjs` to return a single item object instead of an array.

### <samp>🤖 Generated by Copilot at 6f6edad</samp>

> _`updateItem` changed_
> _Custom fields and single object_
> _Autumn of Webflow_

### <samp>🤖 Generated by Copilot at 6f6edad</samp>

*  Add `customFields` prop to update-collection-item action to support custom fields in collections ([link](https://github.com/PipedreamHQ/pipedream/pull/6586/files?diff=unified&w=0#diff-33c8179de6a8912d498d78dc14e9ca516da9f59de7a93743a213494dbba45026L7-R7), [link](https://github.com/PipedreamHQ/pipedream/pull/6586/files?diff=unified&w=0#diff-33c8179de6a8912d498d78dc14e9ca516da9f59de7a93743a213494dbba45026R45-R50), [link](https://github.com/PipedreamHQ/pipedream/pull/6586/files?diff=unified&w=0#diff-33c8179de6a8912d498d78dc14e9ca516da9f59de7a93743a213494dbba45026L52-R62))
*  Change response format of updateItem method in webflow.app.mjs to return a single item object instead of an array ([link](https://github.com/PipedreamHQ/pipedream/pull/6586/files?diff=unified&w=0#diff-0cabe665354e1c8728d99ff78a270be709c31ed5e6871dc251ddcb3957d62f20L248-R248), [link](https://github.com/PipedreamHQ/pipedream/pull/6586/files?diff=unified&w=0#diff-33c8179de6a8912d498d78dc14e9ca516da9f59de7a93743a213494dbba45026L7-R7))

### CODE REVIEW

File: components/webflow/actions/update-collection-item/update-collection-item.mjs [hunk 1]
```diff
@@ -42,19 +42,24 @@ export default {
       description: "URL structure of the Item in your site. Note: Updates to an item slug will break all links referencing the old slug.",
       type: "string",
     },
+    customFields: {
+      type: "object",
+      label: "Custom Fields",
+      description: "Add any custom fields that exist in your collection.",
```
[the remark this answers is not in the document]
Comment (dylburger) [reply components/webflow/actions/update-collection-item/update-collection-item.mjs:48]: Changed "that" to "which" to follow restrictive clause rule (Dreyer's English)

File: components/webflow/actions/update-collection-item/update-collection-item.mjs [hunk 2]
```diff
@@ -42,19 +42,24 @@ export default {
...
-        name: this.name,
-        slug: this.slug,
-        _archived: false,
-        _draft: false,
-      },
+      name: this.name,
+      slug: this.slug,
+      _archived: false,
+      _draft: false,
+      ...this.customFields,
```
[the remark this answers is not in the document]
Comment (dylburger) [reply components/webflow/actions/update-collection-item/update-collection-item.mjs:62]: Removed unnecessary comma (Google's Developer docs style guide)

**Comment by vunguyenhung** [comment]
Hi, I have tested this with 2 custom fields and they passed
image
image

Ready for release!

**Review by michelle0927 — APPROVED** [review]

**Pull request merged**